packages/shared/src/components/fields/CustomSwitch.tsx

```diff
@@ -9,8 +9,8 @@ export interface ContentsSwitchProps {
   name: string;
   checked?: boolean;
   onToggle?: () => unknown;
-  leftContent: React.FC<{ className?: string }> | string;
-  rightContent: React.FC<{ className?: string }> | string;
+  leftContent: React.FC<{ className?: string; secondary?: boolean }> | string;
```

Comment:
I think a better property to pass would be `isChecked` since the content can either be something else entirely and not an icon but might still require the value if it is checked or not.

Reply:
Not quite sure I follow, the prop for the icons is actually `secondary` how would we transpile it to that?

Reply:
@sshanzel I changed to ReactNode this way we can keep native implementation on the input side.
Let me know what you think please.

Reply:
That works too. This was a legacy implementation, I'm more than happy to get the FC removed. Thanks!